tests/test_apps/original/RectorCommand-testApply53/src/SomeTest.php

```diff
@@ -27,4 +29,21 @@ public function testRenames(): void
     public function findSomething(Query $query, array $options): Query {
         return $query;
     }
+
+    public function routes(): void
+    {
+        $routes = new RouteBuilder(new RouteCollection(), '/');
+
+        $routes->scope('/api', function ($routes): void {});
+        $routes->scope('/api', ['something'], function ($routes): void {});
+
+        $routes->resources('/api', function ($routes): void {});
+        $routes->resources('/api', ['something'], function ($routes): void {});
+
+        $routes->prefix('/api', function ($routes): void {});
+        $routes->prefix('/api', ['something'], function ($routes): void {});
+
+        $routes->plugin('/api', function ($routes): void {});
```

Comment:
```suggestion
        $routes->plugin('/api', fn ($routes) => null);
```

To cover the short-arrow usage.

Reply:
@markstory In actual code the closure will generally contain multiple lines, so I don't think `fn()` is usable here, unless rector can itself decide between short and normal closure.

Reply:
🤦 good point. I didn't consider that.